Dockerfile

```
@@ -46,6 +46,10 @@ RUN MODE=$([ "$PHP_DEBUG" = "0" ] && echo "production" || echo "development") &&
 	echo "Using $MODE php.ini" && \
 	tar -xOvf /usr/src/php.tar.xz php-$PHP_VERSION/php.ini-$MODE > /usr/local/etc/php/php.ini
 
+RUN if [ "$PHP_DEBUG" = "1" ] ; then pecl install xdebug \
+	&& echo "zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20190902/xdebug.so" >> /usr/local/etc/php/php.ini ; \
```

```suggestion
	&& docker-php-ext-enable xdebug ; \
```
I think this is the recommended extension installation procedure. Documentation is at https://hub.docker.com/_/php under "PECL extensions".

---

Cool; updated